services/brig/src/Brig/Provider/API.hs

```diff
@@ -137,12 +137,12 @@ routesPublic = do
   get "/provider/activate" (continue activateAccountKeyH) $
     accept "application" "json"
       .&> query "key"
-      .&. query "code"
+        .&. query "code"
```

Comment:
This seems wrong, maybe `.&>` and  `.&.` should be at the same fixity?

Reply:
Currently both are `infixr 3`, though it could be possible that Ormolu isn't picking that up?  Prehaps we need to add them to the .ormolu file?

I don't understand why Ormolu wants to indent these lines.  I can't come up with a logic that is consistent with how it handlles (`:` and `++`) or (`.` and `$`) in other places.

That said, I really don't think "ugly" formatting should be a blocker _for this task_.  If we need to change formatter, or add ORMOLU_DISABLE sections, or quiet hlint, or other we can take that up as an effort separate from aligning CI Ormolu with dev environment HLS/Ormolu.